libappstream-glib/as-release.c

```diff
@@ -769,6 +774,67 @@ as_release_set_description (AsRelease *release,
 			     as_ref_string_new (description));
 }
 
+/**
+ * as_release_get_urls:
+ * @release: a #AsRelease instance.
+ *
+ * Gets the URLs set for the application.
+ *
+ * Returns: (transfer none): hash table of URLs
+ *
+ * Since: 0.7.15
+ **/
+GHashTable *
+as_release_get_urls (AsRelease *release)
+{
+	AsReleasePrivate *priv = GET_PRIVATE (release);
+	return priv->urls;
+}
+
+/**
+ * as_release_get_url_item:
+ * @release: a #AsRelease instance.
+ * @url_kind: the URL kind, e.g. %AS_URL_KIND_HOMEPAGE.
+ *
+ * Gets a URL.
+ *
+ * Returns: string, or %NULL if unset
+ *
+ * Since: 0.7.15
+ **/
+const gchar *
+as_release_get_url_item (AsRelease *release, AsUrlKind url_kind)
```

Comment:
Maybe call it just as_release_get_url() ? _item at the end seems unnecessary and just makes it more of a mouthful.

Reply:
I just coped the name (another mistake?) from AsApp. Maybe `as_release_get_url_by_kind`?

Reply:
Sure, works for me!

Reply:
libappstream just calls this `as_release_get_url(rel, kind)` and has a `as_release_get_url(rel, kind, url)`

Reply:
I think as_release_get_url(rel, kind) would be my preference as well, but as_release_get_url_by_kind() works as well.

Reply:
as_release_get_url it is then. thanks.